test/test_psutil.py

```diff
@@ -3094,18 +3104,13 @@ class TestUnicode(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        with tempfile.NamedTemporaryFile() as f:
-            tdir = os.path.dirname(f.name)
-        cls.uexe = os.path.realpath(os.path.join(tdir, "psutil-è.exe"))
-        shutil.copyfile(sys.executable, cls.uexe)
-        if POSIX:
-            st = os.stat(cls.uexe)
-            os.chmod(cls.uexe, st.st_mode | stat.S_IEXEC)
+        cls.uexe = create_temp_executable_file('è')
+        cls.ubasename = os.path.basename(cls.uexe)
+        assert 'è' in cls.ubasename
 
     @classmethod
     def tearDownClass(cls):
-        if not APPVEYOR:
```

Comment:
mmm this was needed because on APPVEYOR sometimes you get "can't delete file used by another process" errors - I would revert this change